ui_curses.c

```diff
@@ -1840,7 +1840,7 @@ static void spawn_status_program(void)
 		free(argv[i]);
 }
 
-static sig_atomic_t ctrl_c_pressed = 0;
+static _Atomic int ctrl_c_pressed = 0;
```

Comment:
There should be a static assertion somewhere here that checks that `_Atomic int` is lock free. Unfortunately clang is bugged and doesn't define the correct macros. We could assert it at runtime but that's probably overkill since `_Atomic int` will probably be lock free everywhere.